docs/src/adoc/index.adoc

```diff
@@ -3375,6 +3375,35 @@ public interface AccountDao extends CrudDao<Account, UUID> {
 <1> Method annotations are not inherited on override, so we must duplicate
     those we want to keep.
 
+== Testing
+
+The `jdbi-testing` artifact provides a link:{jdbidocs}/testing/JdbiRule.html[JdbiRule]
```

Comment:
Should be `jdbi3-testing`?

Reply:
Actually, I'm thinking it should probably be be `jdbi3-junit4`. JUnit 5 is out and is very different.

Reply:
We could do that.  I'd imagined we just mark each of `junit4` and `junit5` and `testng` (or whatever else people contribute) as optional, and we stash it all in one module, otherwise we end up with 5 different artifacts with 1 class each.

Reply:
I'm okay with that too.